plugins/node/opentelemetry-instrumentation-aws-sdk/package.json

```diff
@@ -52,6 +52,7 @@
   },
   "devDependencies": {
     "@aws-sdk/client-dynamodb": "3.85.0",
+    "@aws-sdk/client-kinesis": "3.85.0",
```

Comment:
is it used anywhere? 
seems the tests for kinesis uses v2 (`aws-sdk`) and this dependency is redundant.